### PR TITLE
[chore] Setup unit testing

### PR DESCRIPTION
**Pull request by erin-at-work**
### *Summary*

- Init unit testing lib

### *How did you test your changes?*

`yarn test:unit `

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,13 +56,17 @@
     "stream-browserify": "^3.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0",
+    "@babel/core": "^7.17.9",
```
Comment (cb-jake) [inline package.json:59]: can we use `ts-jest` vs introducing babel?

**Review by cb-jake — APPROVED** [review]
lgtm. Might be less overhead to use `ts-jest` vs `babel` but I'm not opposed to using `babel`

File: package.json (hunk 1 above)
Comment (ljharb) [reply package.json:59]: why? babel is a far superior transpiler to tsc, and should always be used exclusively to transpile, including for TS.

Comment (cb-jake) [reply package.json:59]: If thats the case, should we upgrade the build to use babel vs tsc?

Comment (ljharb) [reply package.json:59]: 1000% yes

Comment (erin-at-work) [reply package.json:59]: ty both! will make that update 👌

**Pull request merged**